src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -18,6 +20,8 @@
     public CommandResult execute(Model model, CommandHistory history) {
         requireNonNull(model);
         model.resetData(getEmptyConcierge());
+        EventsCenter.getInstance().post(new DeselectGuestListEvent());
```

Comment:
What if a room list is currently being selected? Will the same effect be achieved?